license-compliance/gradle-license-report/allowed-licenses.json

```diff
@@ -549,6 +549,9 @@
     },
     {
       "moduleLicense": "^MIT OR MPL-2.0$"
+    },
+    {
+      "moduleLicense": "^CDDL\\+GPL License$"
```

Comment:
I think we should make this change a bit differently. We should add a normalization that transforms this license into "CDDL OR GPL" (please find corresponding version for those licenses) and add this normalized combination to allowed list